### PR TITLE
latest work on coach reports

### PR DESCRIPTION
**Pull request by indirectlylit**
browsing topics is now mostly functional

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/channel-list-page/index.vue [hunk 1]
```diff
@@ -49,8 +49,13 @@
...
       reportLink(channelId) {
+        const linkTargets = {
+          [PageNames.RECENT_CHANNELS]: PageNames.RECENT_ITEMS_FOR_CHANNEL,
+          [PageNames.TOPIC_CHANNELS]: PageNames.TOPIC_CHANNEL_ROOT,
+          [PageNames.LEARNER_CHANNELS]: PageNames.LEARNER_CHANNEL_ROOT,
+        };
         return {
-          name: PageNames.RECENT_ITEMS_FOR_CHANNEL,
+          name: linkTargets[this.pageName],
           params: {
```
Comment (DXCanas) [inline kolibri/plugins/coach/assets/src/views/channel-list-page/index.vue:68]: don't we need to send in more params than this? like `topicId` and stuff?

I did it another way in my branch, but this is cleaner, I think

Comment (indirectlylit) [reply kolibri/plugins/coach/assets/src/views/channel-list-page/index.vue:68]: check out the latest coach app.js for the specific parameters that each page takes.

Channel root pages only take channel and class IDs

**Comment by DXCanas** [comment]
@indirectlylit I still don't have write access for some reason 
image

**Comment by indirectlylit** [comment]
strange, you're a collaborator with write access 
image

**Comment by christianmemije** [comment]
@DXCanas Same. It's because it is a protected branch.

**Pull request merged**